docker/licenses/extract-licenses.sh

```diff
@@ -0,0 +1,68 @@
+#!/usr/bin/env bash
+# Copyright 2021 Google LLC All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+set -o errexit
+set -o nounset
+set -o pipefail
+
+append_license() {
+  lib=$1
+  path=$2
+  echo "================================================================================" >> ${TMP_LICENSES}
+  echo "= ${lib} licensed under: =" >> ${TMP_LICENSES}
+  echo >> ${TMP_LICENSES}
+  cat "$path" >> ${TMP_LICENSES}
+  echo >> ${TMP_LICENSES}
+  echo "= ${path} MD5 $(cat "${path}" | md5sum | awk '{print $1}')" >> ${TMP_LICENSES}
+  echo "================================================================================" >> ${TMP_LICENSES}
+  echo >> ${TMP_LICENSES}
+
+}
+
+SRC_ROOT=$(dirname "${BASH_SOURCE}")/../..
+TMP_LICENSES=${SRC_ROOT}/docker/licenses/LICENSE
+
+cd ${SRC_ROOT}
+
+# Please don't change this tmp folder, line 55 is hardcoded to this.
+TMP_FOLDER=/tmp/ggg
+
+echo "Collecting LICENCE files from go modules..."
+go-licenses save "src/go/configmanager/main/server.go" --save_path "${TMP_FOLDER}"
+
+# Copy LICENSE files under external
+for i in $(ls bazel-esp-v2/external/ -1);
+do
+  FILE=bazel-esp-v2/external/$i/LICENSE
+  if [ -f ${FILE} ]; then
+    echo "Copy $FILE"
+    mkdir -p ${TMP_FOLDER}/$i
+    cp ${FILE} ${TMP_FOLDER}/$i/LICENSE
+  fi
+done
+
+# Clear file
+echo > ${TMP_LICENSES}
+
+append_license "ESPv2" "${SRC_ROOT}/LICENSE"
+
+while read -r entry; do
+  # This prefix removal is hardcoded to be /tmp/ggg
+  LIBRARY=${entry#\/tmp\/ggg\/}
```

Comment:
The script is not easy to review. Do you want to just run the script and check in the appended license result in the same change? So we can find out the output result.

I patched and tried and found it didn't remove prefix.

Reply:
The generated LICENSE file is in the PR.  It is closed so it is too large.  You can open it.

Looked at License file.  it work,  "/tmp/ggg" is removed from library name. This is copied from:

```
com_github_googleapis_googleapis licensed un

Reply:
I see. Thanks:)

So why don't we remove prefix for `= /tmp/ggg/com_github_grpc_grpc/LICENSE MD5 3b83ef96387f14655fc854ddc3c6bd57`?